### PR TITLE
Countshake

### PR DESCRIPTION
**Pull request by CabinFeverLive**
used stash of css from circle container checkout to create the CSS for the shaking count as well as provide a unique key id based off date now to ensure that each shake renders based upon react state

### CODE REVIEW

File: src/components/ComboCount/index.js [hunk 1]
```diff
@@ -1,4 +1,17 @@
 import React from 'react';
+<<<<<<< HEAD
```
Comment (mlbaquerizo) [inline src/components/ComboCount/index.js:2]: fix merge conflict

File: src/components/App/App.js [hunk 2]
```diff
@@ -47,11 +49,14 @@ export default () => {
         const userId = authentication.getUserId();
         const isCurrentUser = messageUserId === userId;
 
+        console.log("CLASS IN MESSAGE HANDLER: ", comboCountClass);
```
Comment (mlbaquerizo) [inline src/components/App/App.js:52]: remove console.log

File: src/components/ComboCount/countShake.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    backface-visibility: hidden;
+    perspective: 1000px;
+  }
+
+.comboShakeEnd {
+animation: shake 0.41s cubic-bezier(.36,.07,.19,.97) both;
+transform: translate3d(0, 0, 0);
+backface-visibility: hidden;
+perspective: 1000px;
+}
```
Comment (mlbaquerizo) [inline src/components/ComboCount/countShake.css:13]: One last thing you have to do: `npm install`
I think prettier was supposed to fix the indentation here

File: src/components/ComboCount/index.js (hunk 1 above)
Comment (CabinFeverLive) [reply src/components/ComboCount/index.js:2]: Applied the changes within index.js

File: src/components/App/App.js (hunk 2 above)
Comment (CabinFeverLive) [reply src/components/App/App.js]: removed the console log

File: src/components/ComboCount/index.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import './countShake.css';
 
 const ComboCount = ({ count }) => (
```
Comment (mlbaquerizo) [inline src/components/ComboCount/index.js:5]: Make sure to add the `comboCountClass` prop to the function signature

File: src/components/ComboCount/countShake.css (hunk 3 above)
Comment (CabinFeverLive) [reply src/components/ComboCount/countShake.css:13]: Ran the npm install

File: src/components/ComboCount/index.js (hunk 4 above)
Comment (CabinFeverLive) [reply src/components/ComboCount/index.js:5]: added `comboCountClass` prop to the function signature

**Pull request merged**